### PR TITLE
mod: Update dependencies mainly TF to 0.14

### PR DESCRIPTION
**Pull request by xescugc**
Updated all the dependencies but mainly focusing on TF 0.14

### CODE REVIEW

**Comment by xescugc** [comment]
Bumped to version 0.14.2 as it's the newest version as of now.

File: go.sum [hunk 1]
```diff
@@ -349,30 +343,25 @@ github.com/hashicorp/hcl v0.0.0-20170504190234-a4b07c25de5f/go.mod h1:oZtUIOe8dh
...
 github.com/hashicorp/logutils v1.0.0/go.mod h1:QIAnNjmIWmVIIkWDTG1z5v++HQmx9WQRO+LraFDTW64=
 github.com/hashicorp/mdns v1.0.0/go.mod h1:tL+uN++7HEJ6SQLQ2/p+z2pH24WQKWjBPkE0mNTz8vQ=
 github.com/hashicorp/memberlist v0.1.0/go.mod h1:ncdBp14cuox2iFOq3kDiquKU6fqsTBc3W6JvZwjxxsE=
 github.com/hashicorp/memberlist v0.1.3/go.mod h1:ajVTdAv/9Im8oMAAj5G31PhhMCZJV2pPBoIllUwCN7I=
 github.com/hashicorp/serf v0.0.0-20160124182025-e4ec8cc423bb h1:ZbgmOQt8DOg796figP87/EFCVx2v2h9yRvwHF/zceX4=
 github.com/hashicorp/serf v0.0.0-20160124182025-e4ec8cc423bb/go.mod h1:h/Ru6tmZazX7WO/GDmwdpS975F019L4t5ng5IgwbNrE=
 github.com/hashicorp/serf v0.8.2/go.mod h1:6hOLApaqBFA1NXqRQAsxw9QxuDEvNxSQRwA/JwenrHc=
-github.com/hashicorp/terraform v0.13.5 h1:QgPgb/pOBclUMymDji9255FlseySf9dRcUMzJws9BAQ=
-github.com/hashicorp/terraform v0.13.5/go.mod h1:1H1qcnppNc/bBGc7poOfnmmBeQMlF0stEN3haY3emCU=
+github.com/hashicorp/terraform v0.14.0 h1:s0OivKdekGqireI8vzOXkXUHb0QxNfG8rnGuPS+cQMU=
```
Comment (tormath1) [inline go.sum:357]: `v0.14.0` still there :thinking:

**Comment by xescugc** [comment]
Forgot to ran `go mod tidy`

**Comment by tormath1** [comment]
LGTM, except the TF one no big update. You can RS !

**Comment by xescugc** [comment]
Done

**Review by tormath1 — APPROVED** [review]

**Pull request merged**